tests/installation/bootloader_svirt.pm

```diff
@@ -141,18 +140,13 @@ sub run {
 
     my $hdddir = "$basedir/openqa/${share_factory}hdd $basedir/openqa/${share_factory}hdd/fixed";
     my $size_i = get_var('HDDSIZEGB', '10');
-    foreach my $n (1 .. get_var('NUMDISKS')) {
+    foreach my $n (1 .. get_var('NUMDISKS', 1)) {
```

Comment:
Why is this needed?

Reply:
`NUMDISKS` is a mandatory parameter that will be set by the backend code if not specified elsewhere. In case of svirt, https://github.com/os-autoinst/os-autoinst/blob/master//backend/svirt.pm#L46. But this change does not hurt

Reply:
Ok then.